### PR TITLE
[ticket/13454] Remove unused variables, globals, and parameters

### PR DESCRIPTION
**Pull request by marc1706**
https://tracker.phpbb.com/browse/PHPBB3-14335

### CODE REVIEW

File: phpBB/phpbb/session.php [hunk 1]
```diff
@@ -1458,7 +1458,7 @@ function set_login_key($user_id = false, $key = false, $user_ip = false)
 
 		$sql_ary = array(
 			'key_id'		=> (string) md5($key_id),
-			'last_ip'		=> (string) $this->ip,
+			'last_ip'		=> (string) $user_id,
```
Comment (lavigor) [inline phpBB/phpbb/session.php]: Maybe `$user_ip`?

**Comment by marc1706** [comment]
Ready for review

**Comment by Mauron** [comment]
>_drops the mic_

_picks the mic up_

**Comment by marc1706** [comment]
@Nicofuma updated again

**Comment by Nicofuma** [comment]
Looks like I missed your previous "ready for review"

**Pull request merged**